### ISSUE TITLE
Labor Relations: How to read a google sheet csv into R

### ISSUE THREAD

**Issue by hpblood** [HumanitiesDataAnalysis/Directories#6]

**Comment by hpblood** [comment]
This is potentially a good option: https://googlesheets4.tidyverse.org/

**Comment by bmschmidt** [comment]
Yeah that's a great package, I've been using it a lot lately. It can get a bit buggy when uploading more than 1000 rows, but makes it quite easy to read.
